tasks/push-rpm-manifests-to-pyxis/push-rpm-manifests-to-pyxis.yaml

```diff
@@ -110,8 +110,14 @@ spec:
         elif [[ "$(params.server)" == "stage" ]]
         then
           export PYXIS_GRAPHQL_API="https://graphql-pyxis.preprod.api.redhat.com/graphql/"
+        elif [[ "$(params.server)" == "production-internal" ]]
```

Comment:
line 24 should be updated too (which is copied in the README)

Reply:
fixed